### ISSUE TITLE
About Freecol screen needs updating

### ISSUE THREAD

**Issue by Beep6581** [FreeCol/freecol#22]
The "About Freecol" screen needs updating:
- Still links to SourceForge instead of GitHub.
- Copyright date.

**Comment by HighwayofLife** [comment]
We're still working on what would be required to transition to Github from SF. I'm hoping to do this after the 0.12 release.

**Comment by Beep6581** [comment]
Confirmed, though it still doesn't mention GitHub, which I think would be in your interest as SourceForge carries a stigma.

**Comment by Calebrw** [comment]
Did you rebuild and it not show thr GitHub link? If so, I will look at the code agaun tonight.

**Comment by Beep6581** [comment]
`nightly-2018-04-25-24-g`dd1949222

![screenshot_20180711_185658](https://user-images.githubusercontent.com/5844619/42587841-393c2528-853c-11e8-8a7a-5d581b2ace1d.png)

**Comment by Calebrw** [comment]
I must have been more tired than I thought. Confirmed as fixed.

**Comment by Beep6581** [comment]
@Calebrw confirmed :)

By the way, if you mention an issue in the commit, the commit automatically shows here - very useful to keep a tight link between commits and issues.
e.g.
```
git commit -a -m "Fix GitHub links on AboutPanel, closes #22
```
or when the commit message needs to be longer:
```
git commit -a
<editor opens>
<write commit message, first line 50 chars, empty line, then as long as you want to>
Adds GitHub link to About panel, see issue #22
<save>
```
https://git-scm.com/docs/git-commit#_discussion
https://chris.beams.io/posts/git-commit/#seven-rules
https://help.github.com/articles/closing-issues-using-keywords/
